package.json

```diff
@@ -14,7 +14,9 @@
     "eslintconfig",
     "hapi"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "babel-eslint": "^10.1.0"
```

Comment:
Shouldn't we apply what you suggested in https://github.com/hapijs/eslint-plugin/issues/26 directly?

Reply:
We definitely could. I thought we'd want to do it separately but I'm ok to integrate it in this MR.